packages/documentation-site/patternfly-docs/content/get-started/upgrade.md

```diff
@@ -9,25 +9,32 @@ import { Alert } from '@patternfly/react-core';
 <Alert variant="info" isInline title="Before you upgrade"> Make sure that you have already completed [the PatternFly 5 upgrade](https://www.patternfly.org/get-started/upgrade) before attempting to support PatternFly 6. This will ensure that your product has addressed any necessary changes from our previous release.
 </Alert>
 
-PatternFly 6 introduces exciting new features and functionality, including a visual theme refresh that's created with our new [design token system.](/tokens/about-tokens) To learn about the most significant changes in this release, read our [release highlights](/get-started/release-highlights). A detailed list of all changes can be found in our [major release notes](/get-started/upgrade/release-notes). 
+PatternFly 6 introduces exciting new features and functionality, as described in [our release highlights](/get-started/release-highlights). To support these new offerings, we've introduced a long list of changes, the details of which can be found in our [major release notes](/get-started/upgrade/release-notes). 
 
-This guide outlines the requirements for upgrading your codebase to PatternFly 6 and provides additional resources to support your migration efforts.  
+This guide outlines the steps required to upgrade your codebase to PatternFly 6 and provides additional resources to support your migration efforts.  
 
 ## Get help 
 
 If you need support as you upgrade, the PatternFly team is here to help! Reach out to us on [Slack](https://join.slack.com/t/patternfly/shared_invite/zt-1npmqswgk-bF2R1E2rglV8jz5DNTezMQ) or ask a question on our [GitHub discussion board](https://github.com/orgs/patternfly/discussions). We'll always do our best to answer your questions and connect you with the right people quickly. 
 
 ## How to upgrade
 
-When you upgrade your product to PatternFly 6, several breaking changes will likely be introduced to your product’s codebase. 
+When you upgrade your product to PatternFly 6, several breaking changes will likely be introduced to your product’s codebase. These steps walk you through the migration process and connect you with resources that will help you address these breaking changes.
 
-Where we can, we offer a suite of codemods to simplify and streamline your upgrade process. Instead of requiring you to manually identify all errors and issues in your codebase, you can run our codemods to quickly identify and fix major issues. Keep in mind that some changes will still require manual intervention, but our codemods can automatically fix a large amount of issues and flag any issues that do require manual work.
+### 1. Remove all CSS overrides
+At the start of your upgrade process, remove all of your CSS overrides and see how things look. Then, add back any necessary customizations. Since there often isn't a 1:1 equivalent for every PatternFly 5 style in PatternFly 6, your previous overrides may not even be relevant. 
```

Comment:
I wonder if this should be worded more like a recommendation than an actual first step. Maybe something like this? I'm really curious what others think though. @srambach?

```suggestion
### 1. Remove all CSS overrides
At the start of your upgrade process, we strongly recommend removing all of your CSS overrides and see how things look. Then, add back any necessary customizations. Since there often isn't a 1:1 equivalent for every PatternFly 5 style in PatternFly 6, your previous overrides may no longer be relevant. 
```

Reply:
I like it as an actual step. However, this seems to say to remove the overrides and immediately check to see how things look - so if they do this before running codemods, then they will of course see things that now look broken.

What about something like this?
```
### 1. Remove all CSS overrides
At the start of your upgrade process, we strongly recommend removing all of your CSS overrides temporarily. Once you've run through all the codemods, see how things look. Since there often isn't a 1:1 equivalent for every PatternFly 5 style in PatternFly 6, your previous overrides may no longer be relevant. 
As much as possible, we recommend removing CSS overrides so that your product upgrade experience will be smoother for future releases.
If you need to keep any CSS customizations, follow the instructions in step 3 to make sure you are using the appropriate class names and design tokens.
```

Reply:
@mcoker @srambach ahhh okay, so they can remove the css overrides, _then_ run codemods, and then assess? Or is it more helpful to swap the steps order so that codemods come first, and then removing css overrides?

I like this wording!